interfaces/src/main/java/dotty/tools/dotc/interfaces/CompilerCallback.java

```diff
@@ -0,0 +1,30 @@
+package dotty.tools.dotc.interfaces;
+
+import java.io.File;
+
+/** Set of callbacks called in response to events during the compilation process.
+ *
+ *  You should implement this interface if you want to react to one or more of
```

Comment:
I'm not against changing it, but isn't it nicer to keep the style consistent across Dotty? `javadoc` seems to interpret this style correctly so I don't see any downside.